src/controller/command_api.cpp

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2022 Pionix GmbH and Contributors to EVerest
+#include "command_api.hpp"
+
+#include "fmt/core.h"
+
```

Comment:
You're missing a 
#include <fstream>
here, otherwise this won't compile with clang

Reply:
which include?

Reply:
Seems like github just stripped that from my comment:
```cpp
#include <fstream>
```